src/Microsoft.DocAsCode.HtmlToPdf/CoverPage/CoverPageProcessor.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Seems no need to add `CoverPageProcessor` and `BuildCoverPage` here. We can leverage existing `ConceptualDocumentProcessor` to process `cover.md` like any other normal Markdown files. The only difference is that `cover.md` user `documentType` to use `cover.html.primary.tmpl` instead of `conceptual.html.primary.html`, which `ConceptualDocumentProcessor` already recognized `documentType` to do this.